packages/react-app/src/components/ProfileView/Markets.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Alert from "@mui/material/Alert";
+import { Trans } from "@lingui/macro";
+import { Grid, Skeleton } from "@mui/material";
+import { formatAmount } from "../../lib/helpers";
+import { BoxRow } from "..";
+import { useMarketsFromCreator } from "../../hooks/useMarketsFromCreator";
+import { shortenAddress } from "@usedapp/core";
+
+export function Markets({ creatorId }: { creatorId: string }) {
+    const { data: markets, error, isLoading } = useMarketsFromCreator(creatorId);
```

Comment:
you could add a new `creatorId` param to `useMarkets` and replace `useMarketsFromCreator` with `useMarkets({creatorId})`

Reply:
Thanks for the recommendation. Implemented in 5fedd25f722665d086aeaa80c25f9aa1e439df39